src/EnergyPlus/OutputReports.cc

```diff
@@ -2500,6 +2508,10 @@ void VRMLOut(std::string &PolygonAction, std::string &ColorScheme)
     //  Do all detached shading surfaces first
     for (surf = 1; surf <= TotSurfaces; ++surf) {
         if (Surface(surf).HeatTransSurf) continue;
+        int constrNum = Surface(surf).Construction;
+        if (constrNum > 0) {
+            if (DataHeatBalance::Construct(constrNum).TypeIsAirBoundary) continue;
+        }
```

Comment:
I can't see the rest of the code in here, but I'd imagine constrNum could be marked as `const` or just not used.  It doesn't really matter though, because surely the compiler is smart enough to just eliminate that temporary variable.